backend/auth.py

```diff
@@ -190,11 +182,25 @@ def get(self):
                 # This will generate an exception if the user does not exist, preventing login
                 db.User.select().where(db.User.email == self._get_google_email(user)).get()
```

Comment:
Found this when trying the code out: can you change this to be a comparison to `db.User.identity`.